go.mod

```diff
@@ -234,3 +234,5 @@ replace (
 	// the following version across all dependencies.
 	google.golang.org/grpc => google.golang.org/grpc v1.54.0
 )
+
+replace github.com/cosmos/interchain-security/v3 => github.com/cosmos/interchain-security/v3 v3.3.1-0.20240201164551-bde0df5adfef
```

Comment:
nit: move in the replace block and add a docstring

Reply:
It's not the final version. We'll have to change it at least once. I'll address the nit when we have releases (or candidates cut). Unless you feel strongly about this